### ISSUE TITLE
Can't run demo: not downloading checkpoint.

### ISSUE THREAD

**Issue by P-Light** [facebookresearch/meshrcnn#85]
Hello!
Trying to run:
```
python demo/demo.py \
--config-file configs/pix3d/meshrcnn_R50_FPN.yaml \
--input demo/example.png \
--output output_demo \
--onlyhighest MODEL.WEIGHTS meshrcnn://meshrcnn_R50.pth
```
Getting the error:
```
Traceback (most recent call last):
  File "demo/demo.py", line 222, in <module>
    cfg, vis_highest_scoring=args.onlyhighest, output_dir=os.path.join(args.output, im_name)
  File "demo/demo.py", line 44, in __init__
    self.predictor = DefaultPredictor(cfg)
  File "/opt/conda/lib/python3.6/site-packages/detectron2/engine/defaults.py", line 194, in __init__
    checkpointer.load(cfg.MODEL.WEIGHTS)
  File "/opt/conda/lib/python3.6/site-packages/fvcore/common/checkpoint.py", line 121, in load
    assert os.path.isfile(path), "Checkpoint {} not found!".format(path)
AssertionError: Checkpoint meshrcnn://meshrcnn_R50.pth not found!
```
However, if I define `MODEL.WEIGHTS detectron2://ImageNetPretrained/MSRA/R-50.pkl` then script executed fine.

Also, if I write own script like
```
import meshrcnn.utils.model_zoo as zoo
model = zoo.MeshRCNNHandler()
model._get_local_path("meshrcnn://meshrcnn_R50.pth")
```
then it downloading MODEL.WEIGHTS meshrcnn://meshrcnn_R50.pth.

**Comment by gkioxari** [comment]
Aaaah! It's because detectron2 started using it's own PathManager and PathHandler recently and I should register the MeshRCNNHandler to that one instead of the common one in fcvore. PathHandlers are a bit of a mystery :P 

I am submitting a fix, but if you want to get things going now switch this 

https://github.com/facebookresearch/meshrcnn/blob/908ac99e90174dff4d2c7e4032500fed86439b2a/meshrcnn/utils/model_zoo.py#L3

to `from detectron2.utils.file_io import PathManager, PathHandler`

and it should all be good again!

**Comment by gkioxari** [comment]
Fixed by https://github.com/facebookresearch/meshrcnn/commit/c89886f46a0f02871f3bc83a770d907f41a5624b

**Comment by PrakharPrashantSingh** [comment]
hi,

please look into this issue as well...

command used->
python demo.py --config-file ../configs/COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml --input capture.PNG --opts MODEL.WEIGHTS detectron2://COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x/137849600/model_final_f10217.pkl

error output->
    assert os.path.isfile(path), "Checkpoint {} not found!".format(path)
AssertionError: Checkpoint detectron2://COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x/137849600/model_final_f10217.pkl not found!

so basically program is not able to load the model pkl file, i am on windows, what to do in this scanario...

Thanks

**Comment by SubeBila** [comment]
> hi,
> 
> please look into this issue as well...
> 
> command used-> python demo.py --config-file ../configs/COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml --input capture.PNG --opts MODEL.WEIGHTS detectron2://COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x/137849600/model_final_f10217.pkl
> 
> error output-> assert os.path.isfile(path), "Checkpoint {} not found!".format(path) AssertionError: Checkpoint detectron2://COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x/137849600/model_final_f10217.pkl not found!
> 
> so basically program is not able to load the model pkl file, i am on windows, what to do in this scanario...
> 
> Thanks

Hi, I'm having the same problem as you.
Have you solved the problem nowadays and can you tell me how to solve it. Thank you very much.

**Comment by SubeBila** [comment]
> hi,
> 
> please look into this issue as well...
> 
> command used-> python demo.py --config-file ../configs/COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml --input capture.PNG --opts MODEL.WEIGHTS detectron2://COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x/137849600/model_final_f10217.pkl
> 
> error output-> assert os.path.isfile(path), "Checkpoint {} not found!".format(path) AssertionError: Checkpoint detectron2://COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x/137849600/model_final_f10217.pkl not found!
> 
> so basically program is not able to load the model pkl file, i am on windows, what to do in this scanario...
> 
> Thanks

hi,bro.
I meet this same question.But now,I've managed to solve this problem.
This problem is mainly caused by not being able to find the model parameter file (pkl file).So,you only  need hava this file.
You can download the model file you need from the official website of detectron2 project, and then change the model path in the driver file to the path where you store the model file locally. See the following code
`        cfg.defrost()
        cfg.MODEL.WEIGHTS = '/path/model_weight/COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x/model_final_f10217.pkl'
        cfg.freeze() 
        checkpointer.load(cfg.MODEL.WEIGHTS)`
